### PR TITLE
Use type casting instead of *val() method

### PR DESCRIPTION
**Pull request by MorrisJobke**
Followup to #8061

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

**Review by blizzz — APPROVED** [review]

File: apps/user_ldap/lib/Access.php [hunk 1]
```diff
@@ -751,9 +751,9 @@ private function _createAltInternalOwnCloudNameForGroups($name) {
 		} else {
 			natsort($usedNames);
 			$lastName = array_pop($usedNames);
-			$lastNo = intval(substr($lastName, strrpos($lastName, '_') + 1));
+			$lastNo = (int)substr($lastName, strrpos($lastName, '_') + 1);
```
Comment (ChristophWurst) [inline apps/user_ldap/lib/Access.php]: This feels wrong. Although unrelated to this PR, I don't get why we're casting a string to an int.

**Review by ChristophWurst — APPROVED** [review]
Changes look good

File: apps/user_ldap/lib/Access.php (hunk 1 above)
Comment (MorrisJobke) [reply apps/user_ldap/lib/Access.php]: Because this is the number that was added to the name, I guess ;)

Comment (ChristophWurst) [reply apps/user_ldap/lib/Access.php]: Oh, that makes sense, kind of. Thanks 😉

**Pull request merged**